R/bootstrap_dfm.R

```diff
@@ -32,27 +33,6 @@ bootstrap_dfm.default <- function(x, n = 10, ..., verbose = quanteda_options("ve
     check_class(class(x), "bootstrap_dfm")
 }
 
-#' @noRd
```

Comment:
We discussed this two years ago. Users should pass DFM where documents are sentences. I want to removed `bootstrap_dfm()` entirely because I don't see any use case for me and it causes triggers this type of arguments often.